modules/data-table/test/spec.coffee

```diff
@@ -266,19 +275,16 @@ describe 'data-table', ->
           hx.select(rows[2]).selectAll('td').selectAll('.hx-data-table-cell-key').text().should.eql(headers)
           hx.select(rows[2]).selectAll('td').selectAll('.hx-data-table-cell-value').text().should.eql(['Dan', '41', 'Builder'])
 
-        it 'should show the paginator block', ->
-          container.select('.hx-data-table-pagination-block-hidden').size().should.equal(0)
-
         it 'should not show the paginator', ->
-          container.select('.hx-data-table-multi-page').size().should.equal(0)
+          container.selectAll('.hx-data-table-paginator-visible').size().should.equal(0)
 
         it 'should not show the row per page picker', ->
-          container.select('.hx-data-table-page-size').size().should.equal(1)
-          container.select('.hx-data-table-page-size').classed('hx-data-table-select-page-size').should.equal(false)
+          container.selectAll('.hx-data-table-page-size').size().should.equal(2)
+          container.selectAll('.hx-data-table-page-size-visible').size().should.equal(0)
 
         it 'should show the filter box', ->
-          container.select('.hx-data-table-filter-control').size().should.equal(1)
-          container.select('.hx-data-table-filter-control').classed('hx-data-table-filter-visible').should.equal(true)
+          container.select('.hx-data-table-filter').size().should.equal(1)
+          container.select('.hx-data-table-filter').classed('hx-data-table-filter-visible').should.equal(true)
```

Comment:
The `control` suffix didn't really make sense so I figured I'd change it as the rest of the structure has already changed quite a bit